### ISSUE TITLE
Question with running the code

### ISSUE THREAD

**Issue by Bowen-He** [ku2482/fqf-iqn-qrdqn.pytorch#11]
Hi, I ran the code for basically around 4M steps, and suddenly it stoped training and output the model. Do you have any idea about what's wrong here?

**Comment by ku2482** [comment]
@Bowen-He 
Did the process stop? What kind of message did you see? Which OS did you use? 
Please tell me more about the situation so that I can understand what is going on.

The training code evaluates the performance every 4M steps and it takes some time(around 30min, depending on the environment).
It may be the cause of the problem.

Thanks.

**Comment by Bowen-He** [comment]
@ku2482 
I'm using nohup command to run the code on a remote server with Linux as the operating system. And it actually doesn't give out any message about error, that's why I'm coming to look for your help.
The attachment is the situation with PONG, and all other games may have similar situations here.
And I'm using the hypeparameters you set in the original config file.
By the way, I'm currently just useing CPU to train the agent, does that may have any influence on this issue? 

Thanks for response!
![捕获](https://user-images.githubusercontent.com/71317990/100954169-fd666700-354e-11eb-97fd-cf9ee75e6203.PNG)

**Comment by ku2482** [comment]
@Bowen-He 
I'm sorry, but I don't understand what you want to ask.
What is exactly the issue here?

>By the way, I'm currently just useing CPU to train the agent, does that may have any influence on this issue?

To benchmark algorithms on Atari env (200M steps) needs around 6-8 days on GPU, so it's apparently not for CPU.

Thanks.

**Comment by Bowen-He** [comment]
@ku2482 
Sorry for being not clear. I just want to describe my scenario as I can. 
The process just stops at around 4M steps here. And no more data is written into the summary file. The final size for summay file is aroung 500 KB here.

Thanks for your help!

**Comment by ku2482** [comment]
@Bowen-He 
How did you confirm that the process stopped?
Which exact command did you use?
What were the stdout/stderr messages?

Your information is not enough to debug the issue.
I don't know why you checked the file size. It seems the process is still running, isn't it?

**Comment by ku2482** [comment]
I don't force you to follow this, but it's worth reading this kind of guidelines for your career in tech.
https://developer.mozilla.org/en-US/docs/Mozilla/QA/Bug_writing_guidelines

**Comment by Bowen-He** [comment]
@ku2482 
Thanks for you patience and suggestion! It's really helpful!
I rerun the code and got a report like the attachment here. It's just a permission problem.

The command I was using to run the code is
nohup python -u train_fqf.py --cuda --env_id PongNoFrameskip-v4 --seed 0 --config config/fqf.yaml (without sudo just because I don't have the permission on the server)
![捕获](https://user-images.githubusercontent.com/71317990/101051740-c6c93480-35c0-11eb-8339-4947a0fc1fae.PNG)

**Comment by ku2482** [comment]
@Bowen-He 
I don't think you need sudo.
Make sure you are running the command under a directory of ownership.
If not, please ask someone familiar with your server.

Thanks.
